dataactbroker/handlers/fileHandler.py

```diff
@@ -628,13 +628,16 @@ def publish_fabs_submission(submission):
 
             # This table is big enough that we want to only grab 2 columns
             offices = sess.query(Office.office_code, Office.office_name, Office.sub_tier_code, Office.agency_code,
-                                 Office.financial_assistance_office, Office.funding_office).all()
+                                 Office.financial_assistance_awards_office, Office.contract_funding_office,
+                                 Office.financial_assistance_funding_office).all()
             for office in offices:
                 office_dict[office.office_code] = {'office_name': office.office_name,
                                                    'sub_tier_code': office.sub_tier_code,
                                                    'agency_code': office.agency_code,
-                                                   'financial_assistance_office': office.financial_assistance_office,
-                                                   'funding_office': office.funding_office}
+                                                   'financial_assistance_awards_office':
+                                                       office.financial_assistance_awards_office,
+                                                   'funding_office': (office.contract_funding_office or
```

Comment:
Just so I'm understanding this right, this section of code will most likely be edited in the future to have `funding_office` _just_ be `office.financial_assistance_funding_office`.

Reply:
I'm unclear on that. For now we're just being lenient but I'm not sure which one of the two we'll be using or how it'll be changing. I'm assuming this is the case but until a final decision is made it's just either one.